services/dashboard/views/datastream.qtpl

```diff
@@ -1,17 +1,16 @@
-{% import "sensorbucket.nl/sensorbucket/services/core/devices" %}
-{% import "sensorbucket.nl/sensorbucket/services/core/measurements" %}
+{% import "sensorbucket.nl/sensorbucket/pkg/api" %}
 {% import "time" %}
 
 {% func (p *DatastreamPage) Body() %}
 <header class="flex justify-between items-center">
     <div class="flex gap-3 text-sm my-4 italic items-center" hx-target="main">
         <a href="/overview" class="hover:underline text-sky-600">Devices</a>
         <span>/</span>
-        <a href="/overview/devices/{%dl p.Device.ID%}" class="hover:underline text-sky-600">{%s p.Device.Code %}</a>
+        <a href="/overview/devices/{%dl p.Device.GetId()%}" class="hover:underline text-sky-600">{%s p.Device.Code %}</a>
```

Comment:
why the rename to lowercase Id instead of ID? not sure if we're consistent with this now